lib/solidus_affirm/engine.rb

```diff
@@ -8,5 +8,11 @@ class Engine < Rails::Engine
     config.generators do |g|
       g.test_framework :rspec
     end
+
+    config.autoload_paths += %W(#{config.root}/lib)
+
+    initializer "spree.gateway.payment_methods", after: "spree.register.payment_methods" do |app|
```

Comment:
I find it's best to make the least amount of assertions about external users as possible. In a perfect world requiring a gem just loads the code from the gem. An install generator _or_ readme instructions should be used for invasive configuration. Someone might require this for some models, but not the gateway, they would then have to manually remove it from the payment method array.